ssl/s3_lib.c

```diff
@@ -4723,19 +4723,33 @@ EVP_PKEY *ssl_generate_pkey_group(SSL *s, uint16_t id)
      */
 # ifndef OPENSSL_NO_DH
     if (gtype == TLS_GROUP_FFDHE)
+#  if 0
```

Comment:
#if 0 ??

Reply:
The "#if 0" code is stuff we're going to need, but I can't enable yet because EC/DH keygen is not yet in place. As soon as it is this code should be made active.